pemFioi/quickAlgo/subtask.js

```diff
@@ -176,6 +179,16 @@ var initBlocklySubTask = function(subTask, language) {
       updateScores();
    };
 
+   subTask.reloadFunctions = function(newFunctions) {
+      subTask.gridInfos.includeBlocks.generatedBlocks.quickpi[this.level] = newFunctions;

```

Comment:
On top of what's said in the previous comment : the file is `subtask.js`, which is used by all quickAlgo tasks ; it doesn't make sense to have something specific to quickpi (as in, `quickpi` is hardcoded in this line) here.